src/app/containers/Image/helpers/srcSetHelper.js

```diff
@@ -0,0 +1,19 @@
+const createSrcset = (originCode, locator, originalImageWidth) => {
+  const resolutions = [240, 320, 480, 624, 800];
+  if (originCode === 'pips') {
+    return null;
+  }
+
+  const requiredResolutions = resolutions.filter(
```

Comment:
Great that you've added this. Can you please add unit tests to capture the different scenarios - when the originalImageWidth is smaller/larger than listed resolutions. 

When you're looking at test coverage, you can run `npx jest --env=jsdom --coverage` to get a table of coverage. Our aim is to have 100% on all new code.